src/openzaak/components/besluiten/tests/test_besluitinformatieobjecten.py

```diff
@@ -608,6 +644,12 @@ def setUpClass(cls):
             label="external documents",
             auth_type=AuthTypes.no_auth,
         )
+        Service.objects.create(
+            api_type=APITypes.ztc, api_root="http://testserver/catalogi/api/v1/"
+        )
+        Service.objects.create(
+            api_type=APITypes.ztc, api_root="http://openzaak.nl/catalogi/api/v1/"
+        )
```

Comment:
seeing this repeated over and over makes me think we should perhaps have a factory with a trait instead, so we can do:

```py
ServiceFactory.create(is_ztc=True)
```
and then you can still pass the `api_root` override (but go with a default that's used most of the time)

Reply:
I've followed your previous advice and added creation of necessary Services inside factories with FkorServiceUrl fields. 
Now it should work without setting services for FkorServiceUrl fields beforehand. 
I'll clean test files from `Service.objects.create` now